src/utils/constants/labels/FAQs.js

```diff
@@ -0,0 +1,207 @@
+        [es]: ''
+      },
+      A: {
+        [en]: 'Once you find the facility that you are interested in, click on the listing. '
+        + 'It will take you to a screen with several options: availability, capacity, contact '
+        + '(on some facilities that can list their information), health & safety, and resource '
+        + '& referral agency. Click on contact. If the contact information is present, you can '
+        + 'reach out to the provider directly. If this option is not available, click on the '
+        + 'resource & referral agency tab. These agencies have all the contact information available '
+        + 'and will be able to assist you with your waitlist questions.',
+        [es]: ''
+      }
+    },
+    {
+      Q: {
+        [en]: 'As a parent looking for care, do any of the Child Care Programs on this site have '
+        + 'educational support such as tutoring and help with homework?',
+        [es]: ''
+      },
+      A: {
+        [en]: 'Once you have selected the facility or home that you are interested in, you will '
+        + 'have to contact the provider to discuss what services they provide.',
+        [es]: ''
+      }
+    },
+    {
+      Q: {
+        [en]: 'As a parent looking for care, I qualify for Child Action Subsidized Child Care. '
+        + 'Are there any services that help with child care transportation?',
+        [es]: ''
+      },
+      A: {
+        [en]: 'Once you have selected the facility or home that you are interested in, you will have '
+        + 'to contact the provider to discuss what services they provide.',
+        [es]: ''
+      }
+    },
+    {
+      Q: {
+        [en]: 'As a parent looking for care, what can I expect if I choose to enroll my child in a '
+        + 'licensed child care facility?',
+        [es]: ''
+      },
+      A: {
+        [en]: 'Community Care Licensing offers free online videos that provide information on licensed '
+        + 'child care and what to look for when choosing a licensed child care facility.',
+        [es]: ''
+      }
+    },
+    {
+      Q: {
+        [en]: 'As a parent, I have general questions about licensed child care. Also, where may I '
+        + 'file a complaint if needed?',
+        [es]: ''
+      },
+      A: {
+        [en]: 'Community Care Licensing offers a free video that provides information on how to file '
+        + 'a complaint with licensing. You can also call 1-800-Let-Us-No (or 1-844-538-8766) to file a '
+        + 'complaint regarding a licensed child care facility or to ask general licensing questions.',
+        [es]: ''
+      }
+    }
+  ],
+  PROVIDERS: [
+    {
+      Q: {
+        [en]: 'I am a child care provider and I need to know how can I change my telephone number?',
+        [es]: ''
+      },
+      A: {
+        [en]: 'As a child care provider, please contact your assigned Licensed Program Analyst (LPA) '
+        + 'at your local Regional Office and submit changes. ',
```

Comment:
do you need this trailing space at the end of the string?

Reply:
nope